src/Input.test.jsx

```diff
@@ -0,0 +1,76 @@
+import { render, fireEvent } from '@testing-library/react';
+
+import Input from './Input';
+
+// Input test
+// 목적: 입력값이 있을 때만 onChange hanlder를 호출하는가?
+
+// Page: onChangeTitle, value(infoTitle)
+//  - render Input: 이름, 분류, 주소
+// 1. without input
+//  - not call onChange handler
+// 2. with input
+//  - calls onChange handler
+
+describe('Input', () => {
+  const value = '';
+  const handleChange = jest.fn();
+
+  const renderInput = () => render(
+    <Input
+      value={value}
+      onChangeTitle={handleChange}
+    />,
+  );
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('renders Input', () => {
+    const { getByPlaceholderText } = renderInput();
+
+    expect(getByPlaceholderText('이름')).toBeInTheDocument();
+    expect(getByPlaceholderText('이름').value).toBe(value);
```

Comment:
이건 중복된 테스트라서 위에 코드는 삭제해도 될 것 같아요
```suggestion
    expect(getByPlaceholderText('이름').value).toBe(value);
```

Reply:
넵! 위 부분 반영하겠습니다.